### PR TITLE
Support to beans represented by Annotation Literals

### PR DESCRIPTION
**Pull request by mrgurgel**
Hello Atbash!
I added a simple support to annotation literal selection at test scope.

### CODE REVIEW

File: utils-cdi/src/test/java/be/atbash/util/BeanManagerFake.java [hunk 1]
```diff
@@ -71,6 +86,10 @@ public void registerBean(String beanName, Object instance) {
         registeredBeans.put(beanName, instance);
     }
 
+    public void registerBean(AnnotationLiteral beanName, Object instance) {
+        registeredLiterals.put(beanName, new ArrayList<Object>(Arrays.asList(instance)));
```
Comment (rdebusscher) [inline utils-cdi/src/test/java/be/atbash/util/BeanManagerFake.java]: Collections.singletonList(instance) is preferred over Arrays.asList(instance).

Is the new ArrayList() required? (does the List needs to be modifiable)

File: utils-cdi/src/test/java/be/atbash/util/FakeCDI.java [hunk 2]
```diff
@@ -55,9 +60,13 @@ public BeanManager getBeanManager() {
 
     @Override
     public Instance<Object> select(Annotation... qualifiers) {
-        // TODO
-        throw new UnsupportedOperationException("Not implemented be.atbash.ee.jsf.jerry.util.cdi.FakeCDI.select(java.lang.annotation.Annotation...)");
 
+        if(qualifiers != null && qualifiers.length > 0)
+        {
```
Comment (rdebusscher) [inline utils-cdi/src/test/java/be/atbash/util/FakeCDI.java]: I always put { at the end of the line :)

**Review by rdebusscher — CHANGES_REQUESTED** [review]
Hi Marcio,

Thanks for your contribution. Can you change those little remarks?

Thx

File: utils-cdi/src/test/java/be/atbash/util/BeanManagerFake.java (hunk 1 above)
Comment (mrgurgel) [reply utils-cdi/src/test/java/be/atbash/util/BeanManagerFake.java]: In fact, there's no need for "new ArrayList()"

File: utils-cdi/src/test/java/be/atbash/util/FakeCDI.java (hunk 2 above)
Comment (mrgurgel) [reply utils-cdi/src/test/java/be/atbash/util/FakeCDI.java]: Right, code convention adopted

**Comment by mrgurgel** [comment]
Hi @rdebusscher !
You're welcome!
I did the suggested changes, do you need another pullrequest?

Thanks for this project, it simplifies a lot my test cases.

**Review by rdebusscher — APPROVED** [review]

**Comment by rdebusscher** [comment]
Hi @mrgurgel,

I have merged your PR.

Glad that this project is a help for you!

**Pull request merged**